tests/lib/verifyCode.js

```diff
@@ -123,6 +123,34 @@ define([
             assert.notOk
           );
       });
+
+      test('#verifyEmail with reminder param', function () {
+        var user = 'test6' + new Date().getTime();
+        var email = user + '@restmail.net';
+        var password = 'iliketurtles';
+        var uid;
+
+        return respond(client.signUp(email, password), RequestMocks.signUp)
+          .then(function (result) {
+            uid = result.uid;
+            assert.ok(uid, 'uid is returned');
+
+            return respond(mail.wait(user), RequestMocks.mail);
+          })
+          .then(function (emails) {
+            var code = emails[0].html.match(/code=([A-Za-z0-9]+)/)[1];
+            assert.ok(code, 'code is returned');
+
+            return respond(client.verifyCode(uid, code, { reminder: 'first' }),
```

Comment:
I know this is in hindsight, but would it make sense to be strict on the auth-server for the value of reminder by making the only accepted values, `first` and `second`? Currently, with a [random reminder value](https://github.com/mozilla/fxa-auth-server/blob/master/lib/routes/account.js#L1336), auth-server will still verify account but won't log anything. Likewise, validation could be added on the js-client to ensure only these values are processed.